news/2025/2025-01-13-tournament-committee-applications-now-open.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: Tournament Committee Applications Now Open!
+date: 2025-01-13 15:00:00 +0000
+---
+
+Calling all tournament goers! We're excited to announce that, for the first time ever, formal applications for the [Tournament Committee (TC)](/wiki/People/Tournament_Committee) are now open!
+
+![](/wiki/shared/news/2025-01-13-tournament-committee-applications-now-open/banner.jpg)
+
+We are seeking passionate individuals with significant tournament experience — including players — to help support and improve the community.
+
+## Who are we, and what do we do?
+
+The Tournament Committee is a panel of skilled and established organisers and helpers in the tournament scene. Here is some of what we do:
+
+- **Tournament reviews:** Approving or denying badge support for tournaments and communicating any necessary changes to organisers.
+- **Handling reports and inquiries:** Investigating tournament-related issues reported by the community, assessing the validity of claims, and providing timely and well-considered responses to users. 
+- **Maintaining the [tournament support](/wiki/Tournaments/Official_support) wiki page:** Keeping official rules and documentation up to date.
+- **Mentoring the community:** Guiding newer members of the tournament scene towards making better decisions.
+- **Community leadership:** Acting as a representative and leader within the tournament space.
```

Comment:
this list format is very formal. you could do away with the bolded titles and the verbosity:

```suggestion
- Reviewing tournaments for badge support.
- Handling tournament reports and inquiries from the community.
- Keeping rules up to date in the [tournament support](/wiki/Tournaments/Official_support) wiki page.
- Mentoring event organisers, answering questions and providing resources.
```

however, i don't particularly mind this one as it is

Reply:
I can see how it's formal especially compared to the rest of the lists. I think oxford comma is needed on that last line  as we use it elsewhere in the post.

Reply:
edited, didn't notice that woops